pytracking/evaluation/tracker.py

```diff
@@ -256,6 +256,57 @@ def _store_outputs(tracker_out: dict, defaults=None):
         output['object_presence_score_threshold'] = tracker.params.get('object_presence_score_threshold', 0.55)
 
         return output
+        
+
+    def init_tracker(self, frame, optional_box, debug=None, visdom_info=None, save_results=False):
```

Comment:
don't you prefer to create a new class inheriting from this one? in a way that all of the code in it will be ours.

Reply:
not sure its worth this complication

Reply:
done